kola/harness.go

```diff
@@ -194,7 +196,27 @@ func filterTests(tests map[string]*register.Test, pattern, pltfrm string, versio
 		checkPlatforms = append(checkPlatforms, "qemu")
 	}
 
+	var blacklisted bool
 	for name, t := range tests {
+		// Drop anything which is blacklisted directly or by pattern
+		blacklisted = false
+		for _, bl := range BlacklistedTests {
+			match, err := filepath.Match(bl, t.Name)
+			if err != nil {
+				return nil, err
+			}
+			// If it matched the pattern or the name is exact denote that
+			// this test is blacklisted
+			if match || t.Name == bl {
```

Comment:
Shouldn't need to do a direct comparison here, if it's directly equal `match` will already be true (https://play.golang.org/p/6p8UECMI4r4).